Point_set_shape_detection_3/examples/Point_set_shape_detection_3/efficient_RANSAC_parameters.cpp

```diff
@@ -97,9 +97,30 @@ int main()
   Efficient_ransac::Shape_range::iterator it = shapes.begin();
 
   while (it != shapes.end()) {
-    // Prints the parameters of the detected shape.
-    std::cout << (*it)->info() << std::endl;
-
+    
+    // Get specific parameters depending on detected shape.
+    if (dynamic_cast<Plane*>(it->get()))
+      {
```

Comment:
I think I would use `if (Plane* plane = dynamic_cast<Plane*>(it->get())){ Vector_3 normal = plane->normal`